src/ol_infrastructure/applications/mit_learn/Pulumi.applications.mit_learn.CI.yaml

```diff
@@ -66,7 +66,6 @@ config:
     MITOL_COOKIE_NAME: "learn-ci"
     MITOL_LOG_LEVEL: "INFO"
     MITOL_NEW_USER_LOGIN_URL: "https://ci.learn.mit.edu/onboarding"
-    MITOL_NOINDEX: "true"
```

Comment:
The django server doesn't use this for anything. It is currently being set in `settings.py` but has a default, and
- https://github.com/mitodl/mit-learn/pull/2361
is open to remove it.